File: java/act6/act6_1.java
/*
 * @author Hinay, Anthony John C.
 */

 package act6;


class act6_1{
    public static void main(String[] args) {
        
        // Declare the variables
        int i =0;
        String[] names = {
            "Marygin",
            "Sammy",
            "CJay",             
            "CJoy",             
            "Christle"
        };
        
        // display the output
        for(String name : names){
            System.out.printf("%d. %s\n", i, name); i++;
        }          
    }
}

File: scala/act6_1.scala
object act6_1 {

  def main(args: Array[String]): Unit = {
    var i = 0
    val names: Array[String] = Array(
      "Marygin",
      "Sammy",
      "CJay",
      "CJoy",
      "Christle"
    )

    for(n <- names){
        i += 1
        println(s"$i) $n")
    }
  }

}
